src/llvmheaders.h

```diff
@@ -57,11 +64,13 @@
 #include "llvmheaders_38.h"
 #elif LLVM_VERSION == 39
 #include "llvmheaders_39.h"
+#elif LLVM_VERSION == 60
+#include "llvmheaders_60.h"
 #else
-#error "unsupported LLVM version"
+//#error "unsupported LLVM version"
```

Comment:
Can this be uncommented? Are there any problems that occur if it is?

Reply:
Done.